OpenEdXMobile/AndroidManifest.xml

```diff
@@ -326,8 +328,10 @@
         </receiver>
 
         <!-- Setup Facebook -->
-        <meta-data android:name="com.facebook.sdk.ApplicationId"
-            android:value="${facebookAppId}"/>
+        <meta-data android:name="com.facebook.sdk.ApplicationId" android:value="${facebookAppId}"/>
```

Comment:
It's not working with `facebookAppId`. You can see an error on stacktrace as well. Please use `fbLoginProtocolScheme` instead.